frontend/src/constants.js

```diff
@@ -2,3 +2,7 @@
 export const CODALAB_VERSION = '0.5.11';
 
 export const NAVBAR_HEIGHT = 60;
+
+// Worksheet width
+export const EXPAND_PERCENTAGE = '90%';
```

Comment:
Can you rename these to `EXPANDED_WORKSHEET_WIDTH` and `DEFAULT_WORKSHEET_WIDTH`?